README.md

```diff
@@ -1,56 +1,64 @@
 # 0 A.D. / Pyrogenesis Multiplayer Lobby Setup
 
-This README explains how to setup a custom Pyrogenesis Multiplayer Lobby server that can be used with the Pyrogenesis game.
+This README explains how to setup a custom Pyrogenesis Multiplayer Lobby server that can be used
+with the Pyrogenesis game.
```

Comment:
I suppose engine would be more accurate than game?

Reply:
I believe we should change the wording in general to make it clear that these bots are lobby bots for pyrogenesis (the engine), with 0ad (the game) as its main user. Changing this wording is probably a larger task, so I suggest we move that (and any possibly associated discussion) to a separate PR.